app/Policies/RoomPolicy.php

```diff
@@ -163,6 +198,14 @@ public function manageFiles(?User $user, Room $room)
      */
     public function downloadFile(?User $user, Room $room, RoomFile $roomFile)
     {
-        return $room->owner->is($user) || $roomFile->download === true;
+        if ($roomFile->download === true) {
```

Comment:
Should guests have the permission to download files?

Reply:
Yes, see "Changes" section of this PR description